res/demo/app.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "runtime": {
-        "arguments": "--v=1 --enable-crash-reporting",
```

Comment:
These args would never be applied since the provider starts first, so it seemed safe to remove them.